swagger.yaml

```diff
@@ -532,6 +532,58 @@ paths:
         "default":
           description: Software entry not found
 
+  /users:
```

Comment:
Lets be consistent to the style of this spec, endpoints are wrapped in double quotes